doc-content/enterprise-only/openshift/hacep-server-proc.adoc

```diff
@@ -13,7 +13,7 @@ Alternatively, use a different process to deploy the HA CEP server where you can
 
 .Procedure
 
-.  Download the `{PRODUCT_FILE}-reference-implementation.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
+.  Download the `kogito-1.5.0-decision-services-quickstarts.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
```

Comment:
Are you sure? In CR1, rhdm-7.11.0-openshift-drools-hacep-distribution.zip is in rhdm-7.11.0-reference-implementation.zip.